### ISSUE TITLE
hasComment rule fails for tags

### ISSUE THREAD

**Issue by erwanosouf** [whiteclarkegroup/liquibase-linter#90]
The liquibase xsd specifies that no `comment` can be set with a `tagDatabase `elements.

When activating the hasComment rule, it prevents to use tagDatabase because :
- without `comment `linting fails
- with comment xsd validation fails

Suggested fix : 
com.whiteclarkegroup.liquibaselinter.config.rules.core.HasCommentRuleImpl

```
@Override
public boolean invalid(ChangeSet changeSet) {
    return !isTagChangeset(changeSet) && checkNotBlank(changeSet.getComments());
}

protected boolean isTagChangeset(ChangeSet changeSet) {
    // Iterate over changes and return true if tagDatabase is encountered
}
```

**Comment by davidjgoss** [comment]
This is fixed, we'll get a bugfix release out soon

**Comment by erwanosouf** [comment]
Thanks, let me know when this is released!

**Comment by erwanosouf** [comment]
Hello !

Any chance to have this shipped in a release any time soon ?

Thanks in advance !

**Comment by davidjgoss** [comment]
@erwanosouf sorry, yeah, will try and get a release out in the next day or two

**Comment by davidjgoss** [comment]
Released in 0.5.1:
https://github.com/whiteclarkegroup/liquibase-linter/releases/tag/0.5.1

Available in central now:
https://oss.sonatype.org/#nexus-search;gav~~liquibase-linter~0.5.1~~
